crates/sui-faucet/src/faucet/simple_faucet.rs

```diff
@@ -729,6 +1102,132 @@ mod tests {
         );
     }
 
+    #[tokio::test]
+    async fn test_batch_transfer_interface() {
+        let test_cluster = TestClusterBuilder::new().build().await.unwrap();
+        let context = test_cluster.wallet;
+        let config: FaucetConfig = Default::default();
+        let coin_amount = config.amount;
+        let prom_registry = Registry::new();
+        let tmp = tempfile::tempdir().unwrap();
+        let faucet = SimpleFaucet::new(
+            context,
+            &prom_registry,
+            &tmp.path().join("faucet.wal"),
+            config,
+        )
+        .await
+        .unwrap();
+
+        let amounts = &vec![coin_amount; 1];
```

Comment:
```suggestion
        let amounts = &vec![coin_amount];
```
The `1` is redundant here.